### ISSUE TITLE
Add 'skip to main content' link on homepage and French skip links on all other pages [S]

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#926]
**Done When**
- There is a skip to main content link on all the external facing pages
- Internal pages too?

In English, the only unauthenticated page that needs a 'skip to main content' link is the homepage. All other pages have this.

In French, skip to main content is missing across the site. It reads the English "Skip to main content" with French intonnation.

Ensure it's actually skipping to main content and not going through the banner/navigation

When subnavigation is present, consider having a skip to subnavigation link as well.

https://docs.google.com/document/d/18W-nKAH-54wpNy5XSZeEA31my4XRMxpZRVJ3en4ThAA/edit

https://docs.google.com/document/d/1JaJzEnQ7M6tQRziwEC1q8_UUQBizAdVRtUashXdYZeo/edit
